test/lib/jdk/test/lib/containers/docker/Common.java

```diff
@@ -71,6 +71,14 @@ public static DockerRunOptions newOpts(String imageNameAndTag, String testClass)
         return opts;
     }
 
+    public static DockerRunOptions newOpts(String imageNameAndTag, String testClass, String addJavaOpts) {
```

Comment:
It would be good to avoid code duplication and just e.g. have the no-arg `newOpts` pass `null` for `addJavaOpts`.